### ISSUE TITLE
Right click on the iPad doesn't work

### ISSUE THREAD

**Issue by omireault** [AluminumAngel/gloom#3]
We play Gloomhaven with an iPad, and the right click functionality on the map doesn't work.

**Comment by AluminumAngel** [comment]
Try holding your touch.  Does that effectively work as a right-click?

(Assuming that's not broken, I should do something about the tooltips.)

**Comment by omireault** [comment]
No, all it does is a zoom. I think it might detect the underlying element
and decide it can't get a right click...

**Comment by AluminumAngel** [comment]
I haven't actually tested on iPad, only iPhone. Which browser are you using?

At one point I spent a few days attempting to get full touch support working. It's definitely a challenge. Various browsers and devices handle touch differently. In the end I reverted to default behavior (in which browsers generate mouse events from touch input).
